package.json

```diff
@@ -6,6 +6,7 @@
     "build:webpack-only": "hc-tools next-build-webpack-only",
     "build": "next build",
     "lint": "next-hashicorp lint",
+    "prepare": "simple-git-hooks",
```

Comment:
I'm not sure that it makes sense to run this in `prepare`. Per https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-scripts this would get executed when someone runs `npm install`

Reply:
Or is this to ensure that the git hook gets updated? 💭  :nod:

Reply:
Yep! This ensures the hook gets updated. I saw that we do this in [`react-components`](https://hashicorp-mktg.sourcegraph.com/github.com/hashicorp/react-components/-/blob/package.json?L66:5) and [`web-platform-packages`](https://hashicorp-mktg.sourcegraph.com/github.com/hashicorp/web-platform-packages/-/blob/package.json?L47:5) as well.